packages/server/lib/util/proxy.ts

```diff
@@ -12,7 +12,7 @@ const copyLowercaseEnvToUppercase = (name: string) => {
 }
 
 const normalizeEnvironmentProxy = () => {
-  if (!process.env.HTTPS_PROXY) {
+  if (!process.env.HTTPS_PROXY && typeof process.env.HTTP_PROXY !== 'undefined') {
```

Comment:
let's just use the same falsy check here to synchronize this with the other... less mental overhead

// if we don't have HTTPS_PROXY set AND we have HTTP_PROXY set...
```js
if (!process.env.HTTPS_PROXY && process.env.HTTP_PROXY) { 
```